src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -88,6 +88,7 @@ struct FixedKVNode : ChildableCachedExtent {
   uint16_t capacity = 0;
   parent_tracker_t* my_tracker = nullptr;
   RootBlockRef root_block;
+  uint64_t modifications = 0;
```

Comment:
A bit of commits might be helpful to clarify the intention as the implementations might be scattered, and even be wrong accidentally:
```c++
// for the stable extent, modifications is always 0 (is that correct?)
// it will increase for each transaction-local change, so that modifications can be detected (see BtreeLBAMapping.parent_modifications)
```